COLLABORATOR_GUIDE.md

```diff
@@ -391,6 +391,27 @@ Runtime Deprecations and End-of-life APIs (internal or public) must be
 handled as semver-major changes unless there is TSC consensus to land the
 deprecation as a semver-minor.
 
+Possible reasons for Runtime deprecations can be:
+
+* Node.js will no longer be able to support the API, for example if required
+  code is being removed in an upstream project.
+
+* The API is too hard or impossible to use correctly, increasing the risk of
+  undetected bugs or security issues in existing code.
+
+* Breaking the API makes way for simplification of its implementation that is
+  significant enough to make maintainability of Node.js easier, or enables
+  implementation of features that would otherwise be impossible.
+
+In addition, a Documentation-Only deprecation may be issued if it is desirable
+to recommend a single canonical way out of multiple different ones
+which achieve a particular goal with Node.js.
```

Comment:
This is probably correct in most cases but I would not want to strictly limit it to documentation only. I would rather keep that part open and individually decide what to do.

Reply:
Can we please change that to:

```
Possible reasons for a Documentation-Only deprecation can be:

* It is desirable to recommend a single canonical way out of multiple different ones which achieve a particular goal with Node.js.

* An existing API provides very little value to users and it should not be implemented anymore but keeping it does not hurt either.
```